src/app/components/Book/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+
+import { styles } from './styles';
+
+interface Props {
+  title: string;
+  author: string;
+  image: string;
```

Comment:
I think you can use this for the image, `image: ImageSourcePropType;` import it from `react-native`